cli/cli/commands/engine/restart/restart.go

```diff
@@ -158,9 +167,18 @@ func run(_ context.Context, flags *flags.ParsedFlags, _ *args.ParsedArgs) error
 		return stacktrace.Propagate(err, "An error occurred while getting the Kurtosis engine enclave manager UI domain name using the flag with key '%v'; this is a bug in Kurtosis", domainFlagKey)
 	}
 
+	logRetentionPeriodStr, err := flags.GetString(logRetentionPeriodFlagKey)
+	if err != nil {
+		return stacktrace.Propagate(err, "An error occurred while getting the log retention period string from flag: '%v'", logRetentionPeriodStr)
+	}
+	_, err = time.ParseDuration(logRetentionPeriodStr)
+	if err != nil {
+		return stacktrace.Propagate(err, "An error occurred parsing provided log retention period '%v' into a duration. Ensure the provided value has the proper format. Valid time units are \"ns\", \"us\" (or \"µs\"), \"ms\", \"s\", \"m\", \"h\".", logRetentionPeriodStr)
```

Comment:
oh, not day? what about week?

Reply:
as above - right now only using what golangs `time.ParseDuration` supports but can add custom logic later to allow setting other time periods